extensions/sql-migration/src/api/azure.ts

```diff
@@ -83,6 +91,14 @@ export async function getBlobContainers(account: azdata.Account, subscription: S
 	return blobContainers!;
 }
 
+export async function getMigrationControllers(account: azdata.Account, subscription: Subscription, resourceGroupName: string, regionName: string): Promise<azureResource.MigrationController[]> {
+	const api = await getAzureCoreAPI();
+	let result = await api.getMigrationControllers(account, subscription, resourceGroupName, regionName, true);
```

Comment:
You should be making sure that all these calls are handling error conditions correctly as well. At the very least I'd expect that you print the errors out somewhere when it happens.